src/components/DowntimeWarning/index.tsx

```diff
@@ -28,50 +30,54 @@ const ReadMoreLink = styled(ExternalLink)`
   text-decoration: underline;
 `
 
+function Wrapper({ children }: { children: React.ReactNode }) {
```

Comment:
```suggestion
const Wrapper: React.FC = ({ children }) {
```

if you want to avoid typing children manually